.github/ISSUE_TEMPLATE/kubernetes-support.md

```diff
@@ -24,11 +24,11 @@ This is a collector issue for Kubernetes 1.2x support in KubeOne. The following
 * [ ] Check the Kubernetes changelog to ensure there are no breaking changes and removals
 * [ ] Update the `build` image to update Sonobuoy (and other dependencies if needed)
 * [ ] Update the latest supported Kubernetes version in [the API validation](https://github.com/kubermatic/kubeone/blob/main/pkg/apis/kubeone/validation/validation.go#L40-L41) <!-- (link to the PR) -->
-* [ ] Update [default admission controllers](https://github.com/kubermatic/kubeone/blob/main/pkg/kubeflags/data.go) if needed <!-- (link to the PR) -->
-* [ ] Update `pause` image version in `pkg/apis/kubeone`  if needed <!-- (link to the PR) -->
+* [ ] Update [default admission controllers](https://github.com/kubermatic/kubeone/blob/main/pkg/kubeflags/data.go) if needed. It can be obtained via `docker run --rm registry.k8s.io/kube-apiserver:v${KUBERNETES_VERSION} kube-apiserver -h | grep "admission-control"` <!-- (link to the PR) -->
+* [ ] Update `pause` image version in `pkg/apis/kubeone/helpers.go` if needed. You can use `crane ls registry.k8s.io/pause` to search for newer versions of the pause image. <!-- (link to the PR) -->
```

Comment:
I would prefer keeping the checklist as simple and straightforward as possible. Additional explanations should be ideally made below the checklist.

Reply:
okay. I'll move them into a separate section below the action items.